android/app/src/main/java/com/woowacourse/ody/data/remote/core/repository/DefaultJoinRepository.kt

```diff
@@ -1,10 +1,13 @@
 package com.woowacourse.ody.data.remote.core.repository
 
-import com.woowacourse.ody.data.remote.core.entity.join.request.JoinRequest
-import com.woowacourse.ody.data.remote.core.entity.meeting.response.MeetingResponse
+import com.woowacourse.ody.data.remote.core.entity.join.request.toJoinRequest
+import com.woowacourse.ody.data.remote.core.entity.meeting.response.toMeeting
 import com.woowacourse.ody.data.remote.core.service.JoinService
+import com.woowacourse.ody.domain.model.Meeting
+import com.woowacourse.ody.domain.model.MeetingJoinInfo
 import com.woowacourse.ody.domain.repository.ody.JoinRepository
 
 class DefaultJoinRepository(private val service: JoinService) : JoinRepository {
-    override suspend fun postMates(joinRequest: JoinRequest): Result<MeetingResponse> = runCatching { service.postMates(joinRequest) }
+    override suspend fun postMates(meetingJoinInfo: MeetingJoinInfo): Result<Meeting> =
+        runCatching { service.postMates(meetingJoinInfo.toJoinRequest()).toMeeting() }
```

Comment:
```suggestion
    override suspend fun postMates(meetingJoinInfo: MeetingJoinInfo): Result<Meeting> {
        return runCatching { service.postMates(meetingJoinInfo.toJoinRequest()).toMeeting() }
        }
```
오 한줄로 된 함수인데 다음줄에 작성해도 린트 안 걸리나요? ㅋㅋㅋ 신기.... 
개인적으로 위와 같은 방식이 더 좋긴 합니다. 너무 길어지면 return으로 명시해주는 편인데, 그냥 제 취향이라 반영하지 않으셔도 됩니다.

Reply:
더 좋은것 같네요 변경하겠습니다.